packages/styles/src/variables/components/_topic-teaser.scss

```diff
@@ -9,6 +9,12 @@ $content-cols: (
   xl: 7,
 );
 
+$content-gap: (
+  xs: 10px,
```

Comment:
If the gap between the `.topic-teaser-title` and the `.link-list` inside of `.topic-teaser-content` really only should be `10px`, you need to remove the `margin-bottom` from the title.
Otherwise you will have a bigger gap on `xs` breakpoint:
![image](https://github.com/swisspost/design-system/assets/9716662/db34eb69-0238-423e-9429-79c04ff2d0ae)
blue: margin from title
purple: gap

Reply:
After double checking the gap or rather the space in between the 2 elements is supposed to be 24px wich is exactly the size of the title margin. Therefore I removed the gap for xs and sm completely.